app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,6 @@
+<center>
```

Comment:
If you're going to use semantic div tags, maybe go with `<footer></footer>` here. If `<center>` is making the text centered or something, I'd recommend using a class instead (Bootstrap conveniently gives you the built-in `text-center` class for precisely this).

So, this code could look like:
```
<footer class="text-center">
  <h4 class="pink-text">
    And though she be but little, she is fierce.<br />
    - Shakespeare
  </h4>
</footer>